react-native-libraries.json

```
@@ -8590,5 +8590,18 @@
     "ios": true,
     "android": true,
     "newArchitecture": true
+  },
+  {
+    "githubUrl": "https://github.com/wix/Detox",
+    "npmPkg": "detox",
+    "nameOverride": "Detox",
+    "examples": [
+      "https://github.com/wix/Detox/tree/master/examples"
+    ],
+    "images": [
+      "https://github.com/wix/Detox/raw/master/docs/img/Detox.gif"
+    ],
+    "ios": true,
+    "android": true
```

```suggestion
    "android": true,
    "dev": true
```

Let's also add `dev` flag.